app/v2/notifications/post_notifications.py

```diff
@@ -355,6 +355,7 @@ def get_precompiled_letter_template(service_id):
         hidden=True,
         subject='Pre-compiled PDF',
         content='',
+        postage='second'
```

Comment:
Let's use SECOND_CLASS here so that the defaults are easier to find

Reply:
done 👍